Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -28,296 +28,297 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
-		00245045D5E45577751253E413EB5A41 /* AWSCancellationToken.m in Sources */ = {isa = PBXBuildFile; fileRef = E391A6CDBB25CD9CD616C8379AC49870 /* AWSCancellationToken.m */; };
```

Comment:
Should we consider onboard [`xUinque`](https://www.mokacoding.com/blog/xunique/) to minimize thrashing in the proj file?